packages/zoltan2/core/cmake/Zoltan2CheckParMETISVersion.cmake

```diff
@@ -20,6 +20,6 @@ FUNCTION(ZOLTAN2_CHECK_PARMETIS_HAS_VERSION_4_0_3  VARNAME)
 
   "
   )
-  SET(CMAKE_REQUIRED_LIBRARIES ParMETIS::all_libs)
+  SET(CMAKE_REQUIRED_LIBRARIES ParMETIS::all_libs MPI::all_libs)
```

Comment:
FYI: The `MPI::all_libs` target is always defined if MPI is enabled.  So as long as this check only runs when MPI is enable, this should be fine.   (Given that the PR builds all passed, including the non-MPI PR build passed, this should be fine.)

Reply:
ParMETIS requires MPI, so I think it's safe